### DISCUSSION TITLE
8Bitdo Ultimate C Wired

### DISCUSSION THREAD

**Discussion by Ellaquin** [cathery/sys-con#315] [Controller requests]
How would I go about making a profile for the 8Bitdo Ultimate C Wired, when I plug it in, the light starts flashing, but it is not recognized. I don't know how the program knows how to apply a custom profile to a different controller, how do I set that up so it will work?

**Comment by Ellaquin** [comment]
I understand how to follow the example.ini, but I do not understand how to debug it when the controller isn't recognized.

**Comment by MonkaMaczna** [comment]
Also an issue with 8Bitdo Ultimate C 2.4 Wireless. Controller connected to dongle, but diagnostics don't show any entries (both DInput and XInput modes). I would say there's high chance it has something to do with compatibility of those controllers in linux, where [there's this solution which cheats OS into thinking it's generic xbox controller with udev rule](https://gist.github.com/ammuench/0dcf14faf4e3b000020992612a2711e2). Don't know how hard would it be, if possible, to implement it for switch (though it looks like it just changes controller's ID).
